contributors/guide/contributor-cheatsheet/README-de.md

```diff
@@ -365,7 +365,7 @@ anderer Beitragenden zu überlassen, die als Reviewer und Approver für den PR z
 [Kubernetes Forum]: https://discuss.kubernetes.io/
 [Pull-Request Prozess]: /contributors/guide/pull-requests.md
 [GitHub Workflow]: /contributors/guide/github-workflow.md
-[Prow]: https://git.k8s.io/test-infra/prow#prow
+[Prow]: http://sigs.k8s.io/prow/pkg#prow
```

Comment:
Similar change for all localisation files

```suggestion
[Prow]: https://sigs.k8s.io/prow/pkg
```

Reply:
Thanks!! fixed in https://github.com/kubernetes/community/pull/7884/commits/fd7bc7aadf60a56b52bfa3993a05a3dd58a0d2d1